src/Wpf.Ui/WpfUi.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace Wpf.Ui
+{
+    public static class WpfUi
+    {
+#if NET48_OR_GREATER
```

Comment:
#if NET48_OR_GREATER || NETCOREAPP3_0_OR_GREATER

Reply:
thanks for the review and the tip, corrected in the new commit